Client/game_sa/CAnimManagerSA.cpp

```diff
@@ -276,6 +274,10 @@ std::unique_ptr<CAnimBlendAssociation> CAnimManagerSA::CreateAnimAssociation(Ass
 
 CAnimManagerSA::StaticAssocIntface_type CAnimManagerSA::GetAnimStaticAssociation(eAnimGroup animGroup, eAnimID animID)
 {
+    // We check the validity of the group, avoid crashes due to an invalid group
+    if (!IsValidGroup(static_cast<uint32_t>(animGroup)))
```

Comment:
According to the new coding guidelines, please change the uint32_t types to std::uint32_t